From: Danny Kukawka <danny.kukawka@web.de>
Date: Wed, 27 Aug 2008 19:24:33 +0000 (+0200)
Subject: use udevadm instead of (deprecated) udevinfo
X-Git-Url: http://gitweb.freedesktop.org/?p=hal.git;a=commitdiff;h=f6af40c1e7ea54e69eb2d259d91dd212065d5f6e

use udevadm instead of (deprecated) udevinfo

Use udevadm instead of (deprecated) udevinfo. Adapted/fixed version
of a patch contributed by Lucas Hazel <lucas@die.net.au> via HAL ML.
---

--- a/hald/linux/blockdev.c
+++ b/hald/linux/blockdev.c
@@ -1770,7 +1770,7 @@ udev_get_device_file_for_sysfs_path (con
         char *ret;
         char *u_stdout;
         int u_exit_status;
-        const char *argv[] = {"/usr/bin/udevinfo", "--root", "--query", "name", "--path", NULL, NULL};
+        const char *argv[] = {"/sbin/udevadm", "info", "--root", "--query", "name", "--path", NULL, NULL};
         GError *g_error;
 
         ret = NULL;
--- a/hald/linux/coldplug.c
+++ b/hald/linux/coldplug.c
@@ -149,8 +149,8 @@ udev_info_to_hotplug_event (const UdevIn
 static gboolean
 hal_util_init_sysfs_to_udev_map (void)
 {
-	char *udevdb_export_argv[] = { "/usr/bin/udevinfo", "-e", NULL };
-	char *udevroot_argv[] = { "/usr/bin/udevinfo", "-r", NULL };
+	char *udevdb_export_argv[] = { "/sbin/udevadm", "info", "-e", NULL };
+	char *udevroot_argv[] = { "/sbin/udevadm", "info", "-r", NULL };
 	int udevinfo_exitcode;
 	UdevInfo *info = NULL;
 	char *p;
